toplevel/coqargs.mli

```diff
@@ -80,4 +80,4 @@ val parse_args : help:Usage.specific_usage -> init:t -> string list -> t * strin
 val error_wrong_arg : string -> unit
 
 val require_libs : t -> (string * string option * bool option) list
-val build_load_path : t -> Loadpath.coq_path list
+val build_load_path : t -> string list * Loadpath.vo_path list
```

Comment:
```suggestion
val build_load_path : t -> CUnix.physical_path list * Loadpath.vo_path list
```
?

Reply:
Done, thanks. [By the way I tried at some point to add a `Path.t` opaque type to Coq, it was not so easy as I expected]